### PR TITLE
Update to mamba 1.5.11

### PR DESCRIPTION
**Pull request by jjerphan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: Miniforge3/construct.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ user_requested_specs:
 {% if name.endswith("pypy3") %}
   - python 3.9.* *_pypy
   - conda >=24.7.1
-  - mamba >=1.5.9
+  - mamba >=1.5.11
```
Comment (hmaarrfk) [inline Miniforge3/construct.yaml:32]: pypy stuff should be left alone.

Comment (jjerphan) [reply Miniforge3/construct.yaml:32]: I have seen that it has been deprecated. Will it be removed on day?

Also, do notice that I am naively trying to update versions; but someone else might be more efficient performing this update.

Comment (hmaarrfk) [reply Miniforge3/construct.yaml:32]: Jan 2025 i think is the plan

**Comment by hmaarrfk** [comment]
you might be hitting the brownout lol.

**Comment by jjerphan** [comment]
Indeed, from [the announcement](https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/):

> Every ten days in November 2024 (2024-11-01, 2024-11-10, 2024-11-20, 2024-11-30)

**Comment by hmaarrfk** [comment]
Ok now you are getting:
```
se exc
conda.exceptions.PackagesNotFoundError: The following packages are not available from current channels:

  - conda==24.9.3

Current channels:
```

**Pull request merged**